modules/channel-web/src/views/lite/components/messages/MessageGroup.tsx

```diff
@@ -1,4 +1,5 @@
 import classnames from 'classnames'
+import { sortBy } from 'lodash'
```

Comment:
`import sortBy from 'lodash/sortBy'` otherwise you add the whole lodash package

Reply:
agreed